### PR TITLE
Improve grammar in FAQs document

### PR DESCRIPTION
**Pull request by saxbophone**
- Re-arrange word order in notes about the `/` suffixes
- Add _the_ to sentences where it's missing
- _on_ an intranet vs _in_ an intranet

### CODE REVIEW

File: en-US/intro/faqs.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ server {
 ##### How do I set up a sub-path with NGINX?
 
 In case you need to use a sub-path for your Gogs instance, you can change your NGINX configuration to the following
-(note to the both suffixes `/` after **location** and **proxy_pass**):
+(note the `/` suffixes after **location** and **proxy_pass**):
```
Comment (Unknwon) [inline en-US/intro/faqs.md]: It is **extremely important** to point out word "both", so this word can not be deleted, please update, thanks!

Comment (saxbophone) [reply en-US/intro/faqs.md]: Thanks @Unknwon, I understand. I will update it soon.

**Comment by saxbophone** [comment]
@Unknwon Updated :ok_hand:

**Comment by Unknwon** [comment]
Thank you!

**Comment by saxbophone** [comment]
You're welcome! Glad to help :)

**Pull request merged**